shipLHC/AdvTarget.h

```diff
@@ -29,6 +29,8 @@ class AdvTarget : public FairDetector
     void SetConfPar(TString name, Float_t value) { conf_floats[name] = value; }
     void SetConfPar(TString name, Int_t value) { conf_ints[name] = value; }
     void SetConfPar(TString name, TString value) { conf_strings[name] = value; }
+    /** mean position of strips associated with readout channel **/
```

Comment:
Shouldn't it be then end-points, not the mean? Would be good to also mention whether this is opposite corners, centre line or other.